schema/transport.js

```diff
@@ -42,7 +47,7 @@ module.exports = {
 				maxLength: 16
 			}
 		},
-		required: ['port', 'version', 'nonce', 'nethash', 'height']
+		required: ['port', 'httpPort', 'version', 'nonce', 'nethash', 'height', 'os']
```

Comment:
Please review this change carefully. Peers running with old code is not sending httpPort for handshake, and so they can be blocked with this condition.